src/Controller/HomeController.php

```diff
@@ -2,21 +2,21 @@
 
 namespace App\Controller;
 
-use App\Repository\ExhibitionRepository;
+use App\Repository\PresentationExhibitionRepository;
 use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
 use Symfony\Component\HttpFoundation\Response;
 use Symfony\Component\Routing\Annotation\Route;
 
 class HomeController extends AbstractController
 {
     #[Route('/', name: 'app_home')]
-    public function index(ExhibitionRepository $exhibitionRepository): Response
+    public function index(PresentationExhibitionRepository $presexhibition): Response
     {
-        $exhibitions = $exhibitionRepository->findAll();
+        $presexhibition = $presexhibition->findAll();
```

Comment:
No.

First name your repository clearly.
Then do not override a repository with a collection returned by it => create a new variable, it's really not that expensive ...